lint/lint.go

```diff
@@ -32,7 +32,7 @@ var checkFunctions = map[string]struct {
 	"builtin-shadow":    {new: builtinShadowCheck},
 	"range-expr-copy":   {new: rangeExprCopyCheck},
 	"stddef":            {new: stddefCheck},
-	"ptr-to-ref-param":  {new: ptrToRefTypeParamCheck},
+	"ptr-to-ref-param":  {new: ptrToRefParamCheck},
```

Comment:
The other way is to make rule name `ptr-to-ref-type-param`, but I would prefer shorter (current) form.